client/src/client_proxy.rs

```diff
@@ -951,6 +1067,25 @@ impl ClientProxy {
         Ok(req)
     }
 
+    /// Craft an unsigned transaction
+    pub fn create_unsigned_transaction(
```

Comment:
It looks like you're really only using this at one place.  It seems a little unnecessary to have this wrapper function

Reply:
deleted this wrapper function